src/journal/models.py

```diff
@@ -683,3 +684,29 @@ def setup_default_form(sender, instance, created, **kwargs):
             )
 
             default_review_form.elements.add(main_element)
+
+
+def article_remove_from_issue(sender, **kwargs):
+    """
+    When an article is removed from an issue this signal will delete any
+    orderings for that article in the issue.
+    """
+    issue = kwargs.get('instance')
+    action = kwargs.get('action')
+
+    if issue and action == 'post_remove':
```

Comment:
Looks like we need to check the `reverse` argument as well: 
https://docs.djangoproject.com/en/2.1/ref/signals/#m2m-changed
Otherwise, if `ArticleOrdering` with `pk=1` is removed from `Article` with `pk=2` this signal will be invoked and the ArticleOrdering with `pk=2` will be accidentally deleted

Reply:
So basically the instance could be either a `submission.models.Article` or a `journal.models.Issue` depending on which direction the removal has hapenned (issue removes article or article removes issue)